### PR TITLE
Created batches to help get people playing Storybro easier

### PR DESCRIPTION
**Pull request by Warmacha**
Added batches to repository and instructions to the README for windows users to be able to get Storybro up and running easier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ This fork aims to:
 
 To play the game locally, it is recommended that you have an nVidia GPU with 12 GB or more of memory, and CUDA installed. If you do not have such a GPU, each turn can take a couple of minutes or more for the game to compose its response. To install and play locally:
 
+Windows Users:
+```
```
Comment (dustinlacewell) [inline README.md]: No reason to have the following text in quoted block. Just use a normal markdown list.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ This fork aims to:
...
+1: If you do not already have Chocolatey then go ahead and run Install_choco.bat as ADMINISTRATOR and wait for it to finish
+2: Run Install_win afterwards as ADMINISTRATOR, at the end it will automatically launch itself into the storybro shell
+3: Now type in 'storybro models get models_v5' without quote's. allow that to install then type in 'storybro play' to play!
+4: You can quickly access the shell by opening the OpenShell.bat
+
+Note: The reason chocolatey is a seperate install is due to some instances of CMD not working correctly with choco until it has been resetted.
```
Comment (dustinlacewell) [inline README.md]: Just change step to say "Close this terminal and open a new one to ensure Chocolately is ready to go." or similar.

File: OpenShell.bat [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@ECHO off
+
+TITLE StoryMaster Community Pro Super Deluxe 2020: Bro edition
```
Comment (dustinlacewell) [inline OpenShell.bat]: Fix this, :joy:

**Review by dustinlacewell — CHANGES_REQUESTED** [review]
Left some comments in review.

File: README.md (hunk 2 above)
Comment (Warmacha) [reply README.md]: For step 3?

**Pull request merged**